src/ui/qgsprojectpropertiesbase.ui

```diff
@@ -175,7 +175,7 @@
          </item>
          <item>
           <property name="text">
-           <string>Data Sources</string>
+           <string>Data sources</string>
```

Comment:
Any particular reason for this change? see https://github.com/qgis/QGIS-Documentation/pull/2008/files#diff-12068c4d0795351ff6ae31f9af8b5eb6R18

Reply:
Thanks, I thought it was the reverse. I more to fix so.

Reply:
Ok, I fixed all items in QGIS Settings, Project settings, and Vector/Raster layer settings ;-)